src/gui/shellwidget/shellwidget.cpp

```diff
@@ -1007,7 +1007,9 @@ QVariant ShellWidget::TryGetQFontFromDescription(const QString& fdesc) const noe
 	for (const auto& attr : attrs) {
 		if (attr.size() >= 2 && attr[0] == 'h') {
 			bool ok{ false };
-			qreal height = attr.midRef(1).toFloat(&ok);
+			// TODO: the additional toString() call is needed to be compatible with
```

Comment:
What is still left `TODO` here?

Should we file an Issue and reference it here so the fix-up isn't lost to the void?

Reply:
I've updated the comment.

The extra toString() call is needed when Qt < 5.15, but it can be removed for Qt 5.15 or Qt6. Once we the minimum supported Qt version reaches 5.15 this can be removed.